report.Rmd

```diff
@@ -8,9 +8,19 @@ output:
         theme: yeti
 ---
 
-# Introduction
+# Loading Data
+
+```{r Loading Data}
+link_to_data <- "https://osf.io/download/uh4z7/?view_only=de730bd958ef4711819216d30361c8d8"
+file_to_save <- "data_DMDDG.xls"
 
+if (!file.exists(file_to_save)) download.file(url = link_to_data, destfile = file_to_save)
+df <- read_xls(file_to_save)
+
+head(df)
+```
```

Comment:
It cannot be here before there is no library loaded needed to load data. Please move that part were it was.